src/components/molecules/Visualizer/Block/Text/index.tsx

```diff
@@ -137,7 +145,17 @@ const TextBlock: React.FC<Props> = ({
   );
 };
 
-const Wrapper = styled(Border)<{ isTemplate: boolean }>`
+const Wrapper = styled(Border)<{
+  isTemplate: boolean;
+  paddingTop?: number;
+  paddingBottom?: number;
+  paddingLeft?: number;
+  paddingRight?: number;
+}>`
+  padding-top: ${({ paddingTop }) => paddingTop + "px"};
```

Comment:
@mimoham24 Since this could become `undefined` + a `string`, that might give an error. Most places that are like this we have written with a template literal:
```
padding-top: ${({ paddingTop }) => `${paddingTop} + px`};
```